CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Unreleased section should follow [Release Toolkit](https://github.com/newrelic/r
 
 ## Unreleased
 
+## v1.17.0 - 2024-05-15
+### 🐞 Bug fixes
```

Comment:
This is not following the contributing rules:
https://github.com/newrelic/newrelic-prometheus-configurator/blob/main/CONTRIBUTING.md#pull-requests

Reply:
resolved